README.md

```diff
@@ -27,63 +47,99 @@ You will need to perform the following on your development machine:
+
+**Please follow the README of each package you intend to work with on how to get set up and their custom scripts.** However, the above scripts can be run from the root directory in the following manner.
+
+### Package names
+Each package is named with the `deriv-` prefix, however for the scripts above, you do not need to add the `deriv-` prefix as the scripts already prefix the 1st argument of the script with `deriv-`.
+
+You can find the names of packages by first navigating to the `packages` folder. Each subfolder is a package, and contains a `package.json` file. The value of the `name` key in `package.json` is the package name.
 
 ### Usage
+#### Starting a Development Server
+
+If you wish to work on Core, simply run `npm run serve core`.
+
+But for working on any of the other packages (such as Trader, Bot, P2P), perform the following:
+1. Open 2 terminals.
+2. Run `npm run serve {package name}` in the first one. e.g.: `npm run serve translations`, `npm run serve bot`, etc.
+3. Then run `npm run serve core` in the second one.
+
 #### How to Clean Packages
-If you intend to remove `node_modules` folder(s) from the projects, please run `lerna clean` and follow the instructions.
+If you intend to remove `node_modules` folder(s) from the projects, please run `npm run clean` from the root of the project.
 
-You can read more on the various lerna commands (and the `clean` command) over at the [Lerna docs](https://github.com/lerna/lerna/).
+This runs `lerna clean && rm -rf $(git rev-parse --show-toplevel)/node_modules` under the hood.
+You can read more on the various lerna commands (and the [`clean` command](https://github.com/lerna/lerna/tree/master/commands/clean#readme)) over at the [Lerna docs](https://github.com/lerna/lerna/).
 
-#### Examples
-In order to run the `start` script for all packages (`trader`, `bot`, etc.), simply `cd` to the root of the repo and run:
+#### Examples of Script Usage
+If a script supports the "Package param", you can supply a `{package name}` for it to run the script in. At the moment, only 1 package name can be given to a script, if you wish to run in multiple, please use the `lerna` command that's used under the hood as per its docs.
+
+✅ In order to run the `start` script for the `bot` package, simply run:
 ```bash
-npm run start
+npm run start bot
 ```
 
-If you intend to run the script for a specific package, simply run:
-
+✅ Likewise for `trader` (or any other package) with a different script:
 ```bash
-npm run start trader
+npm run test:stylelint trader
 ```
 
-Likewise, to run any of the `deploy` scripts such as `deploy:folder` or `deploy:staging` for a specific package, just run:
+❌ This will not work as the script `deploy:clean` does not support "Package param" (refer to the table in [Working With This Repo](#working-with-this-repo)):
 ```bash
-npm run deploy:folder trader br_test_folder
+npm run deploy:clean bot
```

Comment:
This point is really confusing to me, if we are going to deploy everything, do we still need to run `npm run deploy:clean bot`? Doesn't it also deploy core and trader with itself?

Reply:
i think this should be `npm run deploy:clean` only, without package param?

Reply:
@reza-fs @mustofa-binary You're absolutely correct :D But if you look 2 lines above it, that's an explanation of what **not** to do :D

I added this bit because I kept noticing people adding package name to deploy scripts (or any script that doesn't take in the "package param" as specified in the README scripts table).